src/components/common/DropdownComponent.vue

```diff
@@ -1,11 +1,94 @@
 <template>
-  <div>Dropdown Component</div>
+  <div class="dropdown">
+    <button class="dropdown-button" @click="toggleDropdown">
+      Dropdown List
+      <span class="arrow" :class="{ open: isOpen }"></span>
+    </button>
+
+    <div v-if="isOpen" class="dropdown-menu">
+      <a href="#" class="dropdown-item">option 1</a>
```

Comment:
Please apply loop to display dropdown options instead of hardcoding it.
We can define sample data like https://github.com/Admin-Portal-VDS/Admin-Portal/pull/17/files#diff-585b3b137d429176209062d09ba870fa78176a939cf55b589d24cd8a09836fd0 and display in loop.

https://github.com/Admin-Portal-VDS/Admin-Portal/pull/17/files#diff-520ee50a30fb6db444023af8b5fc9bea0987e04198d29ec9e8609ea21f3bf65eR6-R11